recipe/patches/0003-Remove-all-dependencies-from-setup.py.patch

```diff
@@ -7,12 +8,13 @@ Signed-off-by: Jiajun Yao <jeromeyjj@gmail.com>
 ---
  python/setup.py | 61 ++++++++-----------------------------------------
  1 file changed, 9 insertions(+), 52 deletions(-)
```

Comment:
This line should go into the patch message itself, otherwise it'll be lost upon the next `git format-patch`. You can also just add `Co-Authored-By: ...` - the git history will tell the complete story of how you changed the patch

Reply:
I admit I do not use `git format-patch` and had originally removed lines 1-11. Another reviewer requested I leave them.